### ISSUE TITLE
Associate Flutter SDK with cloned GitHub project in WebStorm

### ISSUE THREAD

**Issue by driftavalli** [flutter/flutter-intellij#1053]
Thanks for the feedback! If your issue is related to the Flutter framework itself,
please open an issue at
[github.com/flutter/flutter](https://github.com/flutter/flutter/issues/new).

## Steps to Reproduce
_Please tell us what you were doing and what went wrong_
Clone a project from GitHub
Open project in Webstorm
Receive message "No Flutter SDK configure".  (Webstorm recognizes project as a Flutter project but fails to associate Flutter SDK). To resolve, have to manually specify the path to the SDK for the current project (File > Settings > Languages & Frameworks > Flutter).

## Version info

Please paste the output of running `flutter doctor` here (available from the command
line or from `Tools > Flutter > Flutter Doctor`). It will provide the version of the
Flutter framework as well as of the IntelliJ plugin.
[✓] Flutter (on Linux, channel master)
    • Flutter at /home/driftavalii/workspace/flutter
    • Framework revision 61c56ba15f (5 days ago), 2017-05-17 14:31:55 -0700
    • Engine revision 6fef702cfa
    • Tools Dart version 1.23.0-dev.11.11

[✓] Android toolchain - develop for Android devices (Android SDK 25.0.3)
    • Android SDK at /home/driftavalii/Android/Sdk
    • Platform android-25, build-tools 25.0.3
    • ANDROID_HOME = /home/driftavalii/Android/Sdk
    • Java binary at: /usr/local/android-studio/jre/bin/java
    • Java version: OpenJDK Runtime Environment (build 1.8.0_112-release-b06)

[✓] Android Studio (version 2.3)
    • Android Studio at /usr/local/android-studio
    • Gradle version 3.2
    • Java version: OpenJDK Runtime Environment (build 1.8.0_112-release-b06)

[✓] IntelliJ IDEA Community Edition (version 2017.1)
    • Dart plugin version 171.4424
    • Flutter plugin version 13.0

[✓] WebStorm (version 2017.1)
    • Flutter plugin version 13.1

[✓] Connected devices
    • Android SDK built for x86 64 • emulator-5554 • android-x64 • Android 6.0 (API 23) (emulator)

**Comment by pq** [comment]
Thanks for the feedback @driftavalli.  Improving SDK discovery is something we're actively looking into.

@devoncarew : I'm guessing https://github.com/flutter/flutter-intellij/issues/1039, would address this?

**Comment by devoncarew** [comment]
I don't think so - you'd have needed to have run `flutter packages get` first from the cli. I think the editor inspection should work here. We may not contribute that for small IDEs, or it may not work properly there.

@driftavalli, if you open a dart file, or the pubspec.yaml file, do you see an option show up at the top of the editor to make the module a flutter module?

**Comment by driftavalli** [comment]
Opening a pubspec.yaml file shows only an option to enable dart support: "Dart support is not enabled for the project [Enable Dart support]() [Open Dart settings]()"
